src/ServicePulse.Host/vue/src/components/failedmessages/AllDeletedMessages.vue

```diff
@@ -234,8 +234,8 @@ onUnmounted(() => {
 });
 
 onMounted(() => {
-  let cookiePeriod = cookies.get("all_deleted_messages_period");
-  if (typeof cookiePeriod === "undefined" || cookiePeriod === "") {
+    let cookiePeriod = cookies.get("all_deleted_messages_period");
+    if (!cookiePeriod) {
```

Comment:
```suggestion
  let cookiePeriod = cookies.get("all_deleted_messages_period");
  if (!cookiePeriod) {
```

These shouldn't be indented